### PR TITLE
UX improvements

### PR DESCRIPTION
**Pull request by QlimaxXael**
### New feature:
- Help button in the main activity that display a popup containing tips and tricks for the app. I wrote some but feel free to add as many explanation as you wish!
### Improvements:
- The buttons are no longer gray, rectangular and annoying. They are now white, oval and much better.
- In createTrackActivity2, the "Create!" button has a different background color to make it stand out from the other.
### What remains to do
- I'll use this PR to see where is the code coverage. I managed to test the popup window without making Travis angry, but it may still need a bit more testing

### CODE REVIEW

File: app/src/androidTest/java/ch/epfl/sweng/runpharaa/MainActivityTest.java [hunk 1]
```diff
@@ -106,6 +110,14 @@ public void testOpenFilters() {
                 .check(matches(isDisplayed()));
     }
 
+    @Test
+    public void testOpenAndCloseHelp() {
```
Comment (somecookie) [inline app/src/androidTest/java/ch/epfl/sweng/runpharaa/MainActivityTest.java]: We should ensure that all tests have at least an assert

Comment (QlimaxXael) [reply app/src/androidTest/java/ch/epfl/sweng/runpharaa/MainActivityTest.java]: Well the test open the popup window and close it by clicking on it, I tried to copy how the testOpenFilters() works. Isn't this enough as it passes over all the code needed to display the popup?

Comment (AntonRagot) [reply app/src/androidTest/java/ch/epfl/sweng/runpharaa/MainActivityTest.java]: Maybe check that you arrive back to your initial activity after closing the popup

File: app/src/main/res/drawable/custom_button_shape.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+    <gradient
+        android:startColor="#ffffff"
+        android:endColor="#ffffff"
+        android:angle="45" />
```
Comment (PeterKrcmar0) [inline app/src/main/res/drawable/custom_button_shape.xml]: Having a gradient that changes from white to white is a bit useless ahah you could have used `<solid>` instead ! But having a light gradient on the button could actually be a nice idea.

File: app/src/main/res/values/styles.xml [hunk 3]
```diff
@@ -24,4 +24,24 @@
         <item name="colorAccent">@color/colorAccent</item>
     </style>
 
+    <style name="ButtonStyle" parent="@android:style/Widget.Button">
```
Comment (PeterKrcmar0) [inline app/src/main/res/values/styles.xml]: Nice job for the button styles, it was time they changed !

File: app/src/androidTest/java/ch/epfl/sweng/runpharaa/MainActivityTest.java (hunk 1 above)
Comment (PeterKrcmar0) [reply app/src/androidTest/java/ch/epfl/sweng/runpharaa/MainActivityTest.java]: This is UI testing and there is a check to see if the popup is displayed, so the test seems good enough for me. Plus, there is no activity change so no Intent triggers to check. You could potentially verify that the popup isn't displayed anymore.

Comment (QlimaxXael) [reply app/src/androidTest/java/ch/epfl/sweng/runpharaa/MainActivityTest.java]: I made it checks that the tab layout from the main activity is visible, I hope that Travis will like it. Thanks for the advice!

File: app/src/main/res/drawable/custom_button_shape.xml (hunk 2 above)
Comment (QlimaxXael) [reply app/src/main/res/drawable/custom_button_shape.xml]: That's true, I'm still tweaking this to have a nice effect but it is quite difficult to have something good. I'll change it to a `<solid>` before merging if I do not use the gradient. Thanks for the review!

**Pull request merged**